### ISSUE TITLE
Combine comments and activity tab in the File sidebar

### ISSUE THREAD

**Issue by nickvergessen** [owncloud/activity#504]
- [ ] "List your own actions in the stream" should only affect file actions (add, change, delete, restore, share). Especially comments activities should be visible.
- [ ] Scrolling/show more to retrieve earlier comments: https://github.com/owncloud/activity/issues/413
- [ ] Add an JS event for comments to add the "New comment" box
- [ ] Add an JS event to allow apps to help formatting their activities (e.g. add delete/edit buttons on comments)

- [ ] Make sure the comments app still works, also when activities are disabled :exclamation:

**Comment by PVince81** [comment]
@pmaier1 would move to backlog...

**Comment by pmaier1** [comment]
Yes. Btw. aren't these quite identical https://github.com/owncloud/core/issues/20259 or is this just the 'activity' part of it?

**Comment by PVince81** [comment]
They're similar. I don't remember the differences. Need to take some time to synthesize and/or reconsider the approach.

**Comment by DeepDiver1975** [comment]
if we want a combined view we need a combined api .... one day ...
